src/custom/theme/cowSwapTheme.tsx

```diff
@@ -23,6 +24,7 @@ export * from '@src/theme/components'
 export function colors(darkMode: boolean): Colors {
   return {
     ...colorsUniswap(darkMode),
```

Comment:
why do we need to still inherit from uniswap?

Reply:
base colors inherit from uniswap already, isn't it?

Reply:
That's right, just fixed it !